src/System.IO/tests/StreamReader/StreamReader.CtorTests.cs

```diff
@@ -42,10 +46,97 @@ public static void CreationFromMemoryStreamWithEncodingTrue()
             var ms2 = new MemoryStream();
             ms2.Write(new byte[] { 65, 66, 67, 68 }, 0, 4);
             ms2.Position = 0;
-            var sr2 = new StreamReader(ms2, false);
+            var sr2 = new StreamReader(ms2, true);
```

Comment:
I think this was incorrect before (there was a same test above and also based on the unit test name).